### ISSUE TITLE
Two runUntyped functions are confusing

### ISSUE THREAD

**Issue by czerwinskilukasz1** [xFAANG/askql#326]
![image](https://user-images.githubusercontent.com/12752520/86490008-4929a000-bd66-11ea-9d5b-e27aca374a49.png)

**Comment by czerwinskilukasz1** [comment]
@mhagmajer FYI

**Comment by mhagmajer** [comment]
Good point, we need to figure out good names for the API. Let's first write down what are the current APIs and what they do
